nengo_loihi/loihi_cx.py

```diff
@@ -101,17 +102,25 @@ def add_probe(self, probe):
         assert probe.target is self
         self.probes.append(probe)
 
-    def set_decay_U(self, tau_s, dt):
-        self.decayU[:] = 1 if tau_s == 0 else -np.expm1(-dt/np.asarray(tau_s))
+    def set_decay_U(self, tau_s, dt, default=False, overwrite=False):
```

Comment:
The `default` logic is also a little confusing to me... it feels like if this was a normal default, I would initialize `decayU` as that default, and then use some sentinel value like `None` to say "hey I want to use default value here."

I can see that the default can't really be set because you use the first configured value. That's probably fine, but if there's a reasonable default value that we can use, we should use it.

Then, rather than having a separate `tau_s` and `default` kwargs, would it be possible to pass `tau_s=None` when you want to use the default? I feel like that would be less awkward... Alternatively, since using the default is a no-op if the group has already been configured, why don't we check for that outside of this method and not call this method if we don't need to configure?

Reply:
> since using the default is a no-op if the group has already been configured, why don't we check for that outside of this method and not call this method if we don't need to configure?

I think that's an option, and if you think that's cleaner I'm fine to change. I felt that doing it the way I did was cleaner, because it kept the `decay_U_set` parameter internal (so all interaction can happen through the `set_decay_U` function). But I'm not strongly opinionated on that.

The reason we can't just set the default up top when we create `decayU` is that I feel like the default should be 5 ms, but to figure out the corresponding decay value, you need to know `dt`. Also, it seems like that default is more associated with the builder (where we're setting up the interneurons, since I essentially use it as a default for interneuron connections).

You can't pass `tau_s=None` to use the default, because the point of `default=True` is that you're setting the default (so you need to pass `tau_s` as well).